frontend/src/lib/components/ProjectList.svelte

```diff
@@ -1,42 +1,70 @@
 <script lang="ts">
   import t from '$lib/i18n';
+  import Icon from '$lib/icons/Icon.svelte';
   import { Badge } from './Badges';
   import { getProjectTypeIcon } from './ProjectType';
-  import type { ProjectItem } from './Projects';
+  import type { ProjectItemWithDraftStatus } from './Projects';
 
-  export let projects: ProjectItem[];
+  export let projects: ProjectItemWithDraftStatus[];
 </script>
 
-<div class="grid grid-cols-2 sm:grid-cols-3 auto-rows-fr gap-2 md:gap-4">
+<div class="grid grid-cols-1 xs:grid-cols-2 sm:grid-cols-3 auto-rows-fr gap-2 md:gap-4 justify-items-center">
   {#each projects as project}
-    <a class="card aspect-square bg-base-200 shadow-base-300 group overflow-hidden" href={`/project/${project.code}`}>
-      <div class="bg" style="background-image: url('{getProjectTypeIcon(project.type)}')" />
-      <div class="card-body z-[1]">
-        <h2 class="card-title overflow-hidden text-ellipsis" title={project.name}>
-          <span class="text-primary inline-flex gap-2 items-center">
-            {project.name}
-          </span>
-        </h2>
-
-        <p>{project.code}</p>
-
-        <p>
-          <span class="i-mdi-account text-xl mb-[-4px]" /> {project.userCount}
-        </p>
-
-        <p class="flex items-end">
-          {#if project.lastCommit}
-            {$t('projectlist.last_change', {
-              lastChange: new Date(project.lastCommit),
+    {#if project.isDraft}
+      <div data-sveltekit-preload-data="false" class="draft card aspect-square bg-base-200 shadow-base-300 group overflow-hidden">
+        <div class="bg" style="background-image: url('{getProjectTypeIcon(project.type)}')" />
+        <div class="card-body z-[1]">
+          <h2 class="card-title overflow-hidden text-ellipsis" title={project.name}>
+            <span class="text-primary inline-flex gap-2 items-center">
+              {project.name}
+              <span
+                class="tooltip text-warning text-xl shrink-0 leading-0"
+                data-tip={$t('projectlist.is_draft')}>
+                <Icon icon="i-mdi-script" />
+              </span>
+            </span>
+          </h2>
+          <p>{project.code}</p>
+          <Badge variant="badge-warning" outline>
+            <Icon icon="i-mdi-progress-clock" />
+            Awaiting approval
+          </Badge>
+          <p class="flex items-end">
+            {$t('projectlist.requested', {
+              requested: new Date(project.createdDate),
             })}
-          {:else}
-            <Badge variant="badge-info" outline>
-              {$t('projectlist.no_changes')}
-            </Badge>
-          {/if}
-        </p>
+          </p>
+        </div>
       </div>
-    </a>
+    {:else}
+      <a class="card aspect-square bg-base-200 shadow-base-300 group overflow-hidden" href={`/project/${project.code}`}>
```

Comment:
I think we forgot to remove this at some point. (probably me)
```suggestion
      <a class="card aspect-square bg-base-200 shadow-base-300 overflow-hidden" href={`/project/${project.code}`}>
```